src/Users/README.md

```diff
@@ -123,6 +180,82 @@ if (!$result) {
 }
 ```
 
+### DELETE OWN ACCOUNT
+```php
+$result = \ATDev\RocketChat\Users\User::deleteOwnAccount(hash("sha256","[USER PASSWORD]"));
```

Comment:
We should think about non-static call here too. We know the user instance to delete, so the method should call the api and dispose all the properties on successful call.